cmd/bridge/main.go

```diff
@@ -85,7 +85,7 @@ func main() {
 	fBranding := fs.String("branding", "okd", "Console branding for the masthead logo and title. One of okd, openshift, ocp, online, dedicated, or azure. Defaults to okd.")
 	fCustomProductName := fs.String("custom-product-name", "", "Custom product name for console branding.")
 	fCustomLogoFile := fs.String("custom-logo-file", "", "Custom product image for console branding.")
-	fStatuspageID := fs.String("statuspage-id", "", "The statuspage.io ID.")
+	fStatuspageID := fs.String("statuspage-id", "", "ID of statuspage.io provider.")
```

Comment:
It's not the provider ID, but specifically the page ID. I'd add a similar description as in openshift/api.